tasks/src/reinforcement-learning/about.md

```diff
@@ -27,23 +28,30 @@ You can contribute variants of this task [here](https://github.com/huggingface/h
 **Environment:** The part of the world the agent interacts, comprising everything outside the agent.
 
 
-**State:** It is the summary of information presented as the function of history of actions,observations and rewards . In the case of a video game it can be a frame (a screenshot), in the case of a chess playing agent it can be the board position, in the case of a trading agent it can be the price of a certain stock.
+Observations/States are the information our agent gets from the environment. In the case of a video game, it can be a frame (a screenshot). In the case of the trading agent, it can be the value of a certain stock, etc.

```

Comment:
```suggestion
Observations and states are the information our agent gets from the environment. In the case of a video game, it can be a frame (a screenshot). In the case of the trading agent, it can be the value of a certain stock.

```